elfo-core/src/init.rs

```diff
@@ -29,48 +29,50 @@ use crate::{
 
 type Result<T, E = StartError> = std::result::Result<T, E>;
 
-async fn start_entrypoints(ctx: &Context, topology: &Topology) -> Result<()> {
+async fn start_entrypoints(ctx: &Context, topology: &Topology, is_check_only: bool) -> Result<()> {
     let futures = topology
         .actor_groups()
         .filter(|group| group.is_entrypoint)
-        .map(|group| {
-            let config = Default::default();
-            ctx.request_to(group.addr, UpdateConfig { config })
-                .resolve()
-                .or_else(|err| async move {
-                    match err {
-                        RequestError::Ignored => Ok(Ok(())),
-                        _ => Err(StartError::Other(
+        .map(|group| async move {
+            fn convert_response<R>(
+                group: String,
+                response: Result<Result<(), impl Into<StartErrorReason>>, RequestError<R>>,
+            ) -> Result<()> {
+                match response {
+                    Ok(Ok(())) => Ok(()),
+                    Ok(Err(e)) => Err(StartError {
+                        group,
+                        reason: e.into(),
+                    }),
+                    Err(RequestError::Ignored) => Ok(()),
+                    Err(RequestError::Closed(..)) => Err(StartError {
+                        group,
```

Comment:
We could include the whole `ActorGroup` in here, but I am not sure if this would be actually useful. There is no error recovery to be done, so the message is only for the end user to analyse.